.github/workflows/codecov.yml

```diff
@@ -0,0 +1,2 @@
+- name: Upload coverage reports to Codecov
```

Comment:
Remove this file. It is not part of our code base. It will cause your tests to fail as unauthorized access.

Reply:
The codecov documentation says that I have to create a yml file for it to run. Its a needed file to integrate codecov to this project.

Reply:
@palisadoes Its removed. Could you kindly check the changes made?